test/plugin/ckpttimer/ckpttimer.cpp

```diff
@@ -0,0 +1,243 @@
+/* FILE: ckpttimer.cpp
+ * AUTHOR: Rohan Garg
+ * EMAIL: rohgarg@ccs.neu.edu
+ * Copyright (C) 2015 Rohan Garg
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include <stdlib.h>
+#include <unistd.h>
+#include <stdio.h>
+#include <signal.h>
+#include <time.h>
+
+#include "dmtcp.h"
+#include "jassert.h"
+
+#define DEBUG_SIGNATURE "[Ckpttimer Plugin]"
+#ifdef CKPTTIMER_PLUGIN_DEBUG
+# define DPRINTF(fmt, ...) \
+  do { fprintf(stderr, DEBUG_SIGNATURE fmt, ## __VA_ARGS__); } while (0)
+#else
+# define DPRINTF(fmt, ...) \
+  do { } while (0)
+#endif
+
+#define PRINTF(fmt, ...) \
+  do { fprintf(stdout, DEBUG_SIGNATURE fmt, ## __VA_ARGS__); } while (0)
+
+#define handleError(msg) \
+    do { perror(msg); exit(EXIT_FAILURE); } while (0)
+
+
+#define CLOCKID         CLOCK_REALTIME
+#define DEFAULT_SIGN    SIGRTMIN+2
```

Comment:
s/DEFAULT_SIGN/DEFAULT_SIG ?
To me, SIGN is short for SIGNATURE, so let's use SIG or SIGNUM or SIGNAL.